apps/frontend/src/components/global/checklist/ChecklistRulesTable.vue

```diff
@@ -0,0 +1,246 @@
+<template>
+  <v-card height="94vh" overflow-auto>
+    <v-card-title class="pt-2">
+      <div
+        style="
+          display: flex;
+          align-items: center;
+          justify-content: space-between;
+          width: 100%;
+        "
+      >
+        <div>
+          <strong>
+            Rules ({{ numItems }} shown, {{ rules.length - numItems }} hidden)
+          </strong>
+        </div>
+        <div style="width: fit-content">
+          <v-switch
+            input-value="true"
+            style="margin-top: 0; padding-top: 0"
+            dense
+            inset
+            label="Short ID"
+            color="teal"
+            hide-details
+            @change="$emit('toggle-short-id')"
+          />
+        </div>
+      </div>
+      <v-spacer class="mt-0 pt-0" />
```

Comment:
feels like this spacer is not necessary when it's size 0

Reply:
Removed